test/scrabble_test.rb

```diff
@@ -5,9 +5,46 @@
 require 'pry'
 
 class ScrabbleTest < Minitest::Test
+
+  def test_it_exists
+    game = Scrabble.new
+
+    assert_instance_of Scrabble, game
+  end
+
   def test_it_can_score_a_single_letter
-    skip
-    assert_equal 1, Scrabble.new.score("a")
-    assert_equal 4, Scrabble.new.score("f")
+    game = Scrabble.new
+
+    assert_equal 1, game.score("a")
+    assert_equal 4, game.score("f")
+    refute_equal 5, game.score("f")
+  end
```

Comment:
Kind of like the fact that this `refute_equal` seems to imply that we're almost openly hostile to any assumptions we might be making in our tests. Again, something that someone else might tell you to take out, but, to me, doesn't hurt.